azurelinuxagent/ga/exthandlers.py

```diff
@@ -223,7 +223,15 @@ def run_status(self):
         return
 
     def get_guid(self, name):
-        return self.last_guids.get(name, None)
+        return self.last_guids.get(name, (None, False))[0]
+
+    def get_log_guid(self, ext_handler):
+        return self.last_guids.get(ext_handler.name, (None, False))[1]
+
+    def set_log_guid(self, ext_handler, log_val):
+        guid = self.get_guid(ext_handler.name)
+        if guid is not None:
+            self.last_guids[ext_handler.name] = (guid, log_val)
```

Comment:
Consider renaming `last_guids` as well, the context is not obvious to the reader

Reply:
fixed.